remote-wallet/src/ledger.rs

```diff
@@ -233,6 +233,19 @@ impl LedgerWallet {
             ver[3].into(),
         ))
     }
+
+    pub fn sign_raw_data(
+        &self,
+        derivation: &DerivationPath,
+        data: &[u8],
+    ) -> Result<Vec<u8>, RemoteWalletError> {
+        let mut payload = get_derivation_path(&derivation);
```

Comment:
Even though this is temporary, this is confusing naming! `get_derivation_path` of something called `derivation` of type `DerivationPath`.

Reply:
Fair, and that part's not temporary. Better?

Reply:
yes, thanks